nengo_viz/static/viz_value.js

```diff
@@ -41,17 +44,17 @@ VIZ.Value = function(parent, sim, args) {
                     .attr('class', 'graph_text unselectable')
                     .attr('y', args.height - (this.margin_bottom-20))
                     .attr('x', args.width - (this.margin_right + 20));
-        
-    this.axis_time_end = axis_time_end[0][0];  
+
+    this.axis_time_end = axis_time_end[0][0];
```

Comment:
And more spacing fixes to remove.  Sorry about this!  The spacing stuff will all get cleared up before the summer school.....   :)